### PR TITLE
11 12 13

### PR DESCRIPTION
**Pull request by dijeth**

### CODE REVIEW

**Comment by jsru** [comment]
Добавляю преподавателя (@ShGKme) для код-ревью.

File: 13-reusability/01-ProgressPlugin/plugins/progress/index.js [hunk 1]
```diff
@@ -10,5 +10,30 @@ export function useProgress() {
...
+  if (router) {
+    router.beforeEach((to) => {
+      progressInstance.start(to.path);
+    });
+    router.afterEach((to) => {
+      progressInstance.finish(to.path);
+    });
+
+    router.onError(() => progressInstance.fail());
+  }
```
Comment (ShGKme) [inline 13-reusability/01-ProgressPlugin/plugins/progress/index.js:26]: An actual error from the router is now lost. It's better to re-throw it here.

**Review by ShGKme — APPROVED** [review]
Approved, but see a comment

File: 13-reusability/01-ProgressPlugin/plugins/progress/index.js (hunk 1 above)
Comment (dijeth) [reply 13-reusability/01-ProgressPlugin/plugins/progress/index.js:26]: Ups, I had added re-throw, but then removed

**Pull request merged**